src/card/styled-components.js

```diff
@@ -70,9 +70,9 @@ export const Thumbnail = styled<{}>('img', ({$theme}) => ({
 }));
 
 export const Title = styled<{}>('h1', ({$theme}) => ({
+  fontWeight: 500,
```

Comment:
Since fontWeight is already included in HeadingSmall, I think we can completely eliminate this line.

Reply:
That sounds reasonable to me too. I've just updated.

It does look like this new change broke the build, though? Although the break doesn't seem to have anything to do with this change.